rss_parser/models/atom/entry.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from rss_parser.models import XMLBaseModel
+from rss_parser.models.types.date import DateTimeOrStr
+from rss_parser.models.types.tag import Tag
+from rss_parser.pydantic_proxy import import_v1_pydantic
+
+pydantic = import_v1_pydantic()
+
+
+class RequiredAtomEntryMixin(XMLBaseModel):
+    entry_id: Tag[str] = pydantic.Field(alias="id")
```

Comment:
Same as previois comment, I think we can use `id` (making sure people are not confused with aliases and that the mapping is as 1/1 as we can make it)